torchgen/model.py

```diff
@@ -788,9 +796,42 @@ def __post_init__(self) -> None:
                     f"that don't have matching signatures: {test_sig} != {f.func.signature()}"
                 )
         assert self.functional.func.kind() == SchemaKind.functional
+        assert (
+            not self.functional.is_view_op
+        ), "View operator shouldn't be grouped into NativeFunctionsGroup objects. Did you add an out= variant to a view operator?"
         assert self.out.func.kind() == SchemaKind.out
+
         if self.inplace is not None:
             assert self.inplace.func.kind() == SchemaKind.inplace
+            mutable_inputs = [
+                a
+                for a in f.func.arguments.flat_all
+                if a.annotation is not None and a.annotation.is_write
```

Comment:
arguments should have a synthetic property that lets you access this without having to 'parse' it out every time